keyboards/knops/mini/keymaps/mverteuil/keymap.c

```diff
@@ -0,0 +1,350 @@
+                writePinLow(D7);
+                break;
+            case 2:
+                if (readPin(B7)) {
+                    writePinLow(C6);
+                } else {
+                    writePinLow(C7);
+                }
+                break;
+            case 3:
+                writePinLow(D4);
+                break;
+            case 4:
+                writePinLow(E6);
+                break;
+            case 5:
+                writePinLow(B4);
+                break;
+            case 6:
+                writePinLow(D6);
+                break;
+        }
+    }
+}
+
+void set_layer_led(int layerId, bool layerState) {
+    writePinHigh(D5);
+    writePinLow(B6);
+    writePinHigh(B0);
+
+    if (layerState) {
+        switch (layerId) {
+            case 0:
+                writePinLow(D5);
+                break;
+            case 1:
+                writePinHigh(B6);
+                break;
+            case 2:
+                writePinLow(B0);
+                break;
+            case 3:
+                writePinHigh(D5);
+                writePinLow(B6);
+                writePinHigh(B0);
+                break;
+        }
+    } else {
+        switch (layerId) {
+            case 0:
+                writePinHigh(D5);
+                break;
+            case 1:
+                writePinLow(B6);
+                break;
+            case 2:
+                writePinHigh(B0);
+                break;
+            case 3:
+                writePinLow(D5);
+                writePinHigh(B6);
+                writePinLow(B0);
+                break;
+        }
+    }
+}
+
+/*
+ * This function led_set_layer gets called when you switch between layers.
+ * It allows you to turn on and off leds for each different layer and do
+ * other cool stuff.
+ */
+void led_set_layer(int layer) {
+    switch (layer) {
+        case 0:
+            set_switch_led(1, true);
+            set_switch_led(2, true);
+            set_switch_led(3, true);
+            set_switch_led(4, true);
+            set_switch_led(5, false);
+            set_switch_led(6, false);
+            break;
+
+        case 1:
+            set_switch_led(1, true);
+            set_switch_led(2, true);
+            set_switch_led(3, true);
+            set_switch_led(4, false);
+            set_switch_led(5, true);
+            set_switch_led(6, false);
+            break;
+
+        case 2:
+            set_switch_led(1, true);
+            set_switch_led(2, true);
+            set_switch_led(3, true);
+            set_switch_led(4, false);
+            set_switch_led(5, false);
+            set_switch_led(6, true);
+            break;
+
+        case 3:
+            set_switch_led(1, false);
+            set_switch_led(2, false);
+            set_switch_led(3, true);
+            set_switch_led(4, true);
+            set_switch_led(5, true);
+            set_switch_led(6, true);
+            break;
+    }
+}
+
+void led_init_ports() {
+    writePinHigh(B7);
```

Comment:
```suggestion
```

Reply:
Ahh, I see this is actually missing from the two defaults now, I think I had taken it from https://github.com/qmk/qmk_firmware/blob/master/keyboards/knops/mini/keymaps/knops/keymap.c#L86